### ISSUE TITLE
Bump gem version

### ISSUE THREAD

**Issue by datibbaw** [agaridata/spf-ruby#35]
Since #34 has been merged ~6 months ago, it would be good to have a bumped version of the gem :)

**Comment by Matthiasclee** [comment]
Agree, would be nice to have a newer version pushed. As of now, the published version of the gem gives the following error:
```
warning: Socket.gethostbyname is deprecated; use Addrinfo.getaddrinfo instead.
```
This is currently fixed, but the fix is not published to RubyGems...

**Comment by tedaford** [comment]
I'll look into publishing this, apologies it has been so long.  Permissions in rubygems are a little "snarled" for this repo, so I'll have to put in some effort outside of work to resolve this.

**Comment by tedaford** [comment]
I could easily tag something in Github if that would be helpful to anyone.

**Comment by datibbaw** [comment]
Our company policy "mandates" the use of proper cut versions 😬

**Comment by datibbaw** [comment]
@tedaford sorry to bug you about this, do you have any updates on this? 😸

**Comment by tedaford** [comment]
Not yet, have had some issues with Rubygems itself (and also we have a new baby), I'll ping some people and see if I can proverbially "cut the Gordian knot". Thanks all for your patience.
